spec/features/reverse_chrono_flow_spec.rb

```diff
@@ -0,0 +1,9 @@
+feature ChitterController do
+  scenario 'post a message to Chitter and see it posted' do
+    visit '/flow/new'
```

Comment:
Perhaps a 'web_helper.rb' file could assists with the following lines of test code...
```
visit '/flow/new'
fill_in('message', with: 'Skynet has become self-aware')
click_button('Submit')
```
Usually the 'helper' files would be used for repetitive code and you've only used these lines twice as far as I can see so perhaps at this stage it's not necessary. If you plan to work on this challenge any further then maybe it would become more relevant.

Reply:
Thanks - yes, some helper files and additional tests would be beneficial